packages/ui/src/components/Meter/Meter.tsx

```diff
@@ -0,0 +1,104 @@
+import styled from '@emotion/styled'
+import type { JSX } from 'react'
+import { Text } from '../Text'
+
+const StyledTitle = styled(Text)`
+  display: inline-block;
+  vertical-align: top;
+  line-height: 22px;
```

Comment:
Why do we change this value here? It should be set in Text depending on variant you choose

Reply:
I removed the styled and used a Text as span direclty and much better 👍🏻